Util/src/main/java/io/deephaven/util/mutable/MutableLong.java

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.util.mutable;
+
+/**
+ * Minimal mutable wrapper for a {@code long} value. Loosely based on
+ * {@code org.apache.commons.lang3.mutable.MutableLong}, but without inheriting from {@link Number}, or providing any
+ * overloads that accept {@code Number} or any boxed types.
+ * <p>
+ * Deliberately does not extend {@code Number}, does not implement {@code toString()}/{@code equals}/{@code hashcode()},
+ * or implement {@code Comparable}.
+ */
+public class MutableLong {
```

Comment:
Yeah my pain threshold is usually 3 copies of something before stepping up to tooling for find-and-replace...

I'll write tests for one and duplicate, then make sure the two types are otherwise the same via copy/paste.